carceral-groups-website/src/api/services/MapPointsService.ts

```diff
@@ -123,17 +75,170 @@ const uniqueDocumentTypes = (docs: MapPoint[]) => {
     return uniqueObjects;
 }
 
-const getDocument = (document_id: string) => {
-    const doc = dataset.find((doc) => doc.id === document_id);
-    if(doc === undefined) return null;
+export type Location = {
+    latlngStr: string;
+    label: string;
+}
+
+// TODO: Remove when backend api is connected
+const MARKERLOCATIONS: Location[] = [
+    { latlngStr: '47.299723843258185, -123.17403244585927', label: 'Washington Corrections Center' },
+    { latlngStr: '46.92520017548775, -123.92100101336285', label: 'Stafford Creek Corrections Center' },
+    { latlngStr: '46.07812903003074, -118.35677764875055', label: 'Washington State Penitentiary' },
+    { latlngStr: '46.73098767622705, -117.16580759999998', label: 'Evergreen Newsroom, Murrow Center' },
+    { latlngStr: '47.19686490207005, -122.6579449486688', label: 'McNeil Island Corrections Center' },
+    { latlngStr: '47.61945939065681, -122.359439800832', label: 'Seattle Post Intelligencer building' },
+    { latlngStr: '47.601990221257424, -122.33189048910353', label: 'Smith Tower, ACLU of Washington state office' },
+    { latlngStr: '46.345958963317436, -120.19008248921602', label: 'La Escuelita, Granger, WA' },
+    { latlngStr: '47.65479243631465, -122.30745790445988', label: 'University of Washington' },
+    { latlngStr: '47.60334696496473, -122.30660900064738', label: 'Black Panthers Party Headquarters (second location)' },
+    { latlngStr: '47.61945939065681, -122.359439800832', label: 'Seattle Post Intelligencer building' },
+    { latlngStr: '47.65835361274492, -122.30385550858871', label: 'McMahon Hall, University of Washington' },
+    { latlngStr: '47.246334710589394, -122.45096066974075', label: 'Tacoma Urban League' },
+    { latlngStr: '47.84570689182511, -122.00011294155688', label: 'Monroe Correctional Complex' },
+]
+const GetLocationLabel = (latlngStr: string): string => {
+    const location = MARKERLOCATIONS.find((marker) => marker.latlngStr === latlngStr);
+    return location ? location.label : latlngStr;
+}
 
-    const blobDocument: BlobDocument = {
-        id: doc.id,
-        title: doc.documentDisplayTitle,
-        fileUrl: doc.fileTitle, // TODO: change to actual file url
-        type: doc.documentType
+/**
+ * Retrieves unique document types for a specific geographic location.
+ * 
+ * API Call: GET /GeographicLocations/{id}/DocumentTypes
+ * 
+ * @param {GeographicLocation} geographicLocation - The geographic location object containing latitude and longitude.
+ * @returns {string[]} An array of unique document types available at the specified geographic location.
+ */
+export const getGeographicLocationsDocumentTypes = (geographicLocation: GeographicLocation) => {
+    // TODO: replace with api call START
+    const points = getAllMapPoints();
+    const filteredPoints = points.filter((point) => {
+        const latlng = `${geographicLocation.Longitude}, ${geographicLocation.Latitude}`
```

Comment:
Structured each service call to closely match how backend api would look like to power the leaflet map.